File: moves (from irene)/json-to-geojson.js
var lat=-15.78 //latitude variable
var long=-47.93 //longitude variable
 var data_url="http://api.openweathermap.org/data/2.5/weather?lat="+lat+"&lon="+long+"&appid=111111111111111111111111111";


   function getmyJson(data_url) {
 return JSON.parse($.ajax({
     type: 'GET',
     url: data_url,
     dataType: 'json',
     global: false,
     async:false,
     success: function(data) {

         var outGeoJson = {}
     outGeoJson['properties'] = data
     outGeoJson['type']= "Feature"
     outGeoJson['geometry']= {"type": "Point", "coordinates":
    [data.coord.lon, data.coord.lat]}

    var brasilia = L.geoJson(outGeoJson, { 
     onEachFeature: function (feature, layer)   
            {
            layer.bindPopup("N&#250;mero da RN: "+feature.properties.id+"<br>"+
                            "Localiza&#231;&#227;o: "+feature.properties.name+"<br>"+
                            "longitude: "+feature.geometry.coordinates[0]+"<br>"+
                            "latitude: "+feature.geometry.coordinates[1]
                            );//just to show something in the popup. could be part of the geojson as well!


        }

}).addTo(map);

    return outGeoJson;
     }
 }).responseText);

}
var myweather = getmyJson(data_url);

File: moves (from irene)/json-to-geojson.py
#! usr/bin/env python

from sys import argv
from os.path import exists
import json as json

script, in_file, out_file = "script", "json/full/places.json", "json/full/places.geojson"

data = json.load(open(in_file))

geojson = {
    "type": "FeatureCollection",
    "features": [
     for d in data:
         {
        "type": "Feature",
        "geometry" : {
            "type": "Point",
            "coordinates": [d["lon"], d["lat"]],
            },
        "properties" : d,
     }]
}


output = open(out_file, 'w')
json.dump(geojson, output)

print(geojson)